ci/docker/python-wheel-manylinux-test.dockerfile

```diff
@@ -27,4 +28,5 @@ COPY python/requirements-wheel-test.txt /arrow/python/
 RUN pip install -r /arrow/python/requirements-wheel-test.txt
 
 COPY ci/scripts/install_gcs_testbench.sh /arrow/ci/scripts/
-RUN PYTHON=python /arrow/ci/scripts/install_gcs_testbench.sh default
+
+RUN PYTHON_VERSION=${python} /arrow/ci/scripts/install_gcs_testbench.sh default
```

Comment:
The `manylinux` test step still tries to install `testbench`.

I'm wondering whether this really sets a `PYTHON_VERSION` or it should be defined with an `ARG` before the `RUN` here. I'm not 100% familiar with what Docker does here.

Reply:
Maybe

```suggestion
ENV PYTHON_VERSION=${python}
RUN /arrow/ci/scripts/install_gcs_testbench.sh default
```